### PR TITLE
README added to perf.

### PR DESCRIPTION
**Pull request by amitjamadagni**
README added to perf.

### CODE REVIEW

File: perf/README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+To run the above files, add the Package BenchmarkLite.jl by doing
```
Comment (acroy) [inline perf/README.md]: I would remove the word "above" here and below.

**Comment by acroy** [comment]
Maybe you can mention which test cases are used.

**Comment by acroy** [comment]
This should be Ok to merge.

**Comment by amitjamadagni** [comment]
Merging this.

**Pull request merged**